### PR TITLE
feat: added cpf validator

### PR DESCRIPTION
**Pull request by caroltrindade**
### Description

This PR adds the CPF validator.

### CODE REVIEW

File: __tests__/cpfCnpj/utils/index.test.ts [hunk 1]
```diff
@@ -27,3 +27,23 @@ describe('calcDigitsPositionsCnpj', () => {
...
+    { input: '846273460', output: '8462734606' },
+    { input: '373950500', output: '3739505001' }
+  ]
+  const secondCalc = [
+    { input: '7733315107', output: '77333151077' },
+    { input: '8462734606', output: '84627346069' },
+    { input: '3739505001', output: '37395050010' }
+  ]
+
+  test.each(firstCalc)('Should return cnpj with first digit calculated', (param) => {
```
Comment (mickaelnn) [inline __tests__/cpfCnpj/utils/index.test.ts:43]:
```suggestion
  test.each(firstCalc)('Should return cpf with first digit calculated', (param) => {
```

File: __tests__/cpfCnpj/utils/index.test.ts [hunk 2]
```diff
@@ -27,3 +27,23 @@ describe('calcDigitsPositionsCnpj', () => {
...
+    expect(calcDigitsPositionsCpf(param.input)).toBe(param.output)
+  })
+  test.each(secondCalc)('Should return cnpj with two digits calculated', (param) => {
```
Comment (mickaelnn) [inline __tests__/cpfCnpj/utils/index.test.ts:46]:
```suggestion
  test.each(secondCalc)('Should return cpf with two digits calculated', (param) => {
```

File: src/cpfCnpj/validators/index.ts [hunk 3]
```diff
@@ -13,3 +13,17 @@ export const isCnpj = (cnpj: string): boolean => {
...
 }
+
+export const isCpf = (cpf: string): boolean => {
+  // get first 9 numbers from cpf
+  const firstNumbers = cpf.substr(0, 9)
+  // do first calc
+  const firstCalc = calcDigitsPositionsCpf(firstNumbers)
+  // do second calc init by 10 position
+  const secondCalc = calcDigitsPositionsCpf(firstCalc, 11)
+  // concat the second digit on cnpj
```
Comment (mickaelnn) [inline src/cpfCnpj/validators/index.ts:24]:
```suggestion
  // concat the second digit on cpf
```

**Review by mickaelnn — CHANGES_REQUESTED** [review]

**Review by mickaelnn — APPROVED** [review]
LGTM

File: __tests__/cpfCnpj/validators/index.test.ts [hunk 4]
```diff
@@ -21,3 +21,25 @@ describe('isCnpj', () => {
...
+describe('isCpf', () => {
+  const validCpf = [
+    '77333151077',
+    '84627346069',
+    '37395050010'
+  ]
+  const invalidCpf = [
+    '77333151078',
+    '479980796',
+    '373.950.500-10',
```
Comment (alanneves) [inline __tests__/cpfCnpj/validators/index.test.ts:34]: is this CPF invalid?

**Review by alanneves — CHANGES_REQUESTED** [review]

File: __tests__/cpfCnpj/validators/index.test.ts (hunk 4 above)
Comment (caroltrindade) [reply __tests__/cpfCnpj/validators/index.test.ts:34]: Yes, it has a formatting

Comment (alanneves) [reply __tests__/cpfCnpj/validators/index.test.ts:34]: I'm not sure this is a good approach, our function should be smart enough to identify if it's already formatted.

**Review by alanneves — APPROVED** [review]
LGTM

**Pull request merged**